.gitignore

```diff
@@ -2,3 +2,5 @@ env/
 monorail-key.json
 *.pyc
 __pycache__/
+webapp/node_modules/
+webapp/bower_components/
```

Comment:
Adding this here ought to cause it to not be deployed, can you verify it works with dev_appserver and with a real deploy? I think it worked before because webapp/.gitignore isn't listed in .gcloudignore.

Reply:
I moved the `bower_components` rule back to `webapp/.gitignore`.